manifests/n/NSSM/NSSM/2.24-101-g897c7ad/NSSM.NSSM.locale.en-US.yaml

```diff
@@ -0,0 +1,28 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: NSSM.NSSM
+PackageVersion: 2.24-101-g897c7ad
+PackageLocale: en-US
+Publisher: NSSM
+PublisherUrl: https://nssm.cc/
+PublisherSupportUrl: https://nssm.cc/bugs
+Author: Iain Patterson
+PackageName: NSSM
+PackageUrl: https://nssm.cc/
+License: Public Domain
+LicenseUrl: https://git.nssm.cc/nssm/nssm/src/master/README.txt
```

Comment:
Hi @Fei1Yang,

This link is currently giving a 503 error: 
![image](https://user-images.githubusercontent.com/114952565/220499669-4211f20d-5618-4e79-b600-a89cd9a598a7.png)